### PR TITLE
docs(compute): instances/elastic-metal/dedibox datasheet

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  h1: Instances - Technical product specification
+  paragraph: This page shows the technical characteristics of the Instances offer.
+tags: dedibox server
+dates:
+  validation: 2023-12-18
+  posted: 2023-12-18
+categories:
+  - compute
+---
+
```
Comment (bene2k1) [inline compute/instances/reference-content/instances-datasheet.mdx:15]:
```suggestion

# Instances datasheet
The datasheet for Instances provides a concise overview of their performance, technical features, components, materials, and associated documentation.
```

File: bare-metal/dedibox/reference-content/dedibox-datasheet.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  h1: Dedibox - Technical product specification
+  paragraph: This page shows the technical characteristics of the Dedibox offer.
+tags: dedibox server
+dates:
+  validation: 2023-12-18
+  posted: 2023-12-18
+categories:
+  - compute
+---
+
```
Comment (bene2k1) [inline bare-metal/dedibox/reference-content/dedibox-datasheet.mdx:15]:
```suggestion

# Dedibox datasheet
The datasheet for Dedibox provides a concise overview of their performance, technical features, components, materials, and associated documentation.
```

File: bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+  h1: Elastic Metal - Technical product specification
+  paragraph: This page shows the technical characteristics of the Elastic Metal offer.
+tags: elastic-metal server
+dates:
+  validation: 2023-12-18
+  posted: 2023-12-18
+categories:
+  - bare-metal
+---
+
```
Comment (bene2k1) [inline bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx:15]:
```suggestion

# Elastic Metal server datasheet
The datasheet for Elastic Metal servers provides a concise overview of their performance, technical features, components, materials, and associated documentation.
```

File: bare-metal/dedibox/reference-content/dedibox-datasheet.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: Dedibox - Technical product specification
+  description: This page shows the technical characteristics of the Dedibox offer.
```
Comment (RoRoJ) [inline bare-metal/dedibox/reference-content/dedibox-datasheet.mdx:4]:
```suggestion
  description: This page shows the technical characteristics of the different Dedibox offers.
```

File: bare-metal/dedibox/reference-content/dedibox-datasheet.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+content:
+  h1: Dedibox - Technical product specification
+  paragraph: This page shows the technical characteristics of the Dedibox offer.
```
Comment (RoRoJ) [inline bare-metal/dedibox/reference-content/dedibox-datasheet.mdx:7]:
```suggestion
  paragraph: This page shows the technical characteristics of the different Dedibox offers.
```

File: bare-metal/dedibox/reference-content/dedibox-datasheet.mdx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+tags: dedibox server
+dates:
+  validation: 2023-12-18
+  posted: 2023-12-18
+categories:
+  - compute
+---
+
+# Dedibox datasheet
+The datasheet for Dedibox provides a concise overview of their performance, technical features, components, materials, and associated documentation.
```
Comment (RoRoJ) [inline bare-metal/dedibox/reference-content/dedibox-datasheet.mdx:17]:
```suggestion
This datasheet provides a concise overview of the performance, technical features, components, materials, and associated documentation for different Dedibox offers.
```

File: bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx [hunk 7]
```diff
@@ -0,0 +1,250 @@
+---
+meta:
+  title: Elastic Metal - Technical product specification
+  description: This page shows the technical characteristics of the Elastic Metal offer.
```
Comment (RoRoJ) [inline bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx:4]:
```suggestion
  description: This page shows the technical characteristics of the different Elastic Metal offers.
```

File: bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+content:
+  h1: Elastic Metal - Technical product specification
+  paragraph: This page shows the technical characteristics of the Elastic Metal offer.
```
Comment (RoRoJ) [inline bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx:7]:
```suggestion
  paragraph: This page shows the technical characteristics of the different Elastic Metal offers.
```

File: bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+tags: elastic-metal server
+dates:
+  validation: 2023-12-18
+  posted: 2023-12-18
+categories:
+  - bare-metal
+---
+
+# Elastic Metal server datasheet
+The datasheet for Elastic Metal servers provides a concise overview of their performance, technical features, components, materials, and associated documentation.
```
Comment (RoRoJ) [inline bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx:17]:
```suggestion
This datasheet provides a concise overview of the performance, technical features, components, materials, and associated documentation for the different Elastic Metal server offers.
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 10]
```diff
@@ -0,0 +1,155 @@
+---
+meta:
+  title: Instances - Technical product specification
+  description: This page shows the technical characteristics of the Instances offer.
```
Comment (RoRoJ) [inline compute/instances/reference-content/instances-datasheet.mdx:4]:
```suggestion
  description: This page shows the technical characteristics of the different Instance offers.
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+content:
+  h1: Instances - Technical product specification
+  paragraph: This page shows the technical characteristics of the Instances offer.
```
Comment (RoRoJ) [inline compute/instances/reference-content/instances-datasheet.mdx:7]:
```suggestion
  paragraph: This page shows the technical characteristics of the different Instance offers.
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+tags: dedibox server
+dates:
+  validation: 2023-12-18
+  posted: 2023-12-18
+categories:
+  - compute
+---
+
+# Instances datasheet
+The datasheet for Instances provides a concise overview of their performance, technical features, components, materials, and associated documentation.
```
Comment (RoRoJ) [inline compute/instances/reference-content/instances-datasheet.mdx:17]:
```suggestion
This datasheet provides a concise overview of the performance, technical features, components, materials, and associated documentation for different Instance offers.
```

File: menu/navigation.json [hunk 13]
```diff
@@ -542,6 +542,10 @@
                     "label": "Choosing an Instance type",
```
Comment (RoRoJ) [inline menu/navigation.json:542]: Maybe we can add a link to the Instances datasheet from within this page?

Comment (bene2k1) [reply menu/navigation.json:542]: Link added

File: bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx [hunk 14]
```diff
@@ -0,0 +1,250 @@
...
+## Server EM-I210E-NVME
+
+| Server EM-I210E-NVME  |                                   |
+|-----------------------|-----------------------------------|
+| **CPU**               | AMD EPYC 7313P 16C/32T 3 GHz      |
+| **RAM**               | 128 GB                            |
+| **Disk**              | 2 x 1.92 TB NVMe                  |
+| **Public bandwidth**  | 1 Gbps                            |
+| **Private bandwidth** | 10 Gbps                           |
+| **Availability Zones**| PARIS1, PARIS 2                   |
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/reference-content/elastic-metal-datasheet.mdx:170]:
```suggestion
| **Availability Zones**| PARIS 1, PARIS 2                   |
```

**Review by RoRoJ — APPROVED** [review]

**Comment by bene2k1** [comment]
Please do not merge before ok from @ofranc

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 15]
```diff
@@ -0,0 +1,175 @@
...
+
+<Concept>
+
+## STARDUST1 Instances
+
+ See below the technical specifications of Stardust Instances:
+
+| Range            	  | Learning                                	|
+|:------------------------|:------------------------------------------|
+| Instance Type     	| STARDUST1-S                             	|
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:33]:
```suggestion
| Instance type     	| STARDUST1-S                             	|
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 16]
```diff
@@ -0,0 +1,175 @@
...
+| Availability Zone 	| PAR1, AMS1 and WAW2                     	|
+| Storage           	| Local (10GB) or Block (from 10GB to 10TB) |
+| Max. NIC bandwidth  | 100 Mbit/s              	                |
+| CPU Type          	| AMD EPYC 7282 (2,8 GHz) or 7281 (2,1 GHz)	|
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:37]:
```suggestion
| CPU type          	| AMD EPYC 7282 (2,8 GHz) or 7281 (2,1 GHz)	|
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 17]
```diff
@@ -0,0 +1,175 @@
...
+
+<Concept>
+
+## Developement and General Purpose Instances
+
+See below the technical specifications of Development and General Purpose Instances:
+
+| Range                         	| Cost-Optimized                                                              	|
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | DEV1 <br /> GP1                                                      	        |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:54]:
```suggestion
| Instance type   	              | DEV1 <br /> GP1                                                      	        |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 18]
```diff
@@ -0,0 +1,175 @@
...
+| Availability Zone   	          | PAR1, PAR2, PAR3 (excl. DEV1), AMS1, AMS2, WAW1, WAW2                         |
+| Storage   	                    | Local (up to 150GB) or Block (from 10GB to 10TB)                                            |
+| Max. NIC bandwidth   	          | From 200 to 500 Mbps                                                          |
+| CPU Type   	                    | DEV1: AMD EPYC 7281 (2,1 GHz) or equivalent<br />GP1: AMD EPYC 7410P (2 GHz) or equivalent|
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:58]:
```suggestion
| CPU type   	                    | DEV1: AMD EPYC 7281 (2,1 GHz) or equivalent<br />GP1: AMD EPYC 7410P (2 GHz) or equivalent|
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 19]
```diff
@@ -0,0 +1,175 @@
...
+
+<Concept>
+
+## PLAY2 and PRO2 Instances
+
+See below the technical specifications of PLAY2 and PRO2 Instances:
+
+| Range                         	| Cost-Optimized                                                              	|
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | PLAY2 <br /> PRO2                                                    	        |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:74]:
```suggestion
| Instance type   	              | PLAY2 <br /> PRO2                                                    	        |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 20]
```diff
@@ -0,0 +1,175 @@
...
+| Availability Zone   	          | PAR1, PAR2, PAR3 (excl PLAY2), AMS1, AMS2, AMS3, WAW1, WAW2, WAW3             |
+| Storage   	                    | Block (from 10 GB to 10 TB)                                                  |
+| Max. NIC bandwidth   	          | From 100 Mbps  to 6 Gbps                                                      |
+| CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:78]:
```suggestion
| CPU type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 21]
```diff
@@ -0,0 +1,175 @@
...
+  - compute
+---
+
+# Instances datasheet
+
+This datasheet provides a concise overview of the performance, technical features, components, materials, and associated documentation for different Instance offers.
+
+<Message type="note">
+  Note that the **Max. NIC bandwidth** specifies the maximum available bandwidth for this Instance type. Bandwidth is shared between internet and Private Network traffic.
+
```
Comment (ldecarvalho-doc) [inline compute/instances/reference-content/instances-datasheet.mdx:22]:
```suggestion
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 22]
```diff
@@ -0,0 +1,175 @@
...
+<Concept>
+
+## COP-ARM Instances
+
+The table below displays the technical specifications of COP-ARM Instances:
+
+| Range                         	| Cost-Optimized                                                              	|
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | COPARM1                                                             	        |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:94]:
```suggestion
| Instance type   	              | COPARM1                                                             	        |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 23]
```diff
@@ -0,0 +1,175 @@
...
+| Availability Zone   	          | PAR2                                                                          |
+| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
+| Max. NIC bandwidth   	          | From 200 Mbps to 3.2 Gbps                                                     |
+| CPU Type   	                    | ARM (Ampere Altra Max M128-30)                                                |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:98]:
```suggestion
| CPU type   	                    | ARM (Ampere Altra Max M128-30)                                                |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 24]
```diff
@@ -0,0 +1,175 @@
...
+
+<Concept>
+
+## Enterprise Instances
+
+See below the technical specifications of Enterprise Instances:
+
+| Range                         	| Production-Optimized                                                          |
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | ENT1                                                                	        |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:114]:
```suggestion
| Instance type   	              | ENT1                                                                	        |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 25]
```diff
@@ -0,0 +1,175 @@
...
+| Availability Zone   	          | PAR1, PAR2, PAR3, AMS1, AMS2, AMS3, WAW1, WAW2, WAW3                          |
+| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
+| Max. NIC bandwidth   	          | From 400 Mbps to 20 Gbps                                                      |
+| CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:118]:
```suggestion
| CPU type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 26]
```diff
@@ -0,0 +1,175 @@
...
+
+<Concept>
+
+## Production-Optimized Instances
+
+See below the technical specifications of Production-Optimized Instances:
+
+| Range                         	| Production-Optimized                                                          |
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | POP2                                                                	        |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:137]:
```suggestion
| Instance type   	              | POP2                                                                	        |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 27]
```diff
@@ -0,0 +1,175 @@
...
+| Availability Zone   	          | PAR1, PAR2, PAR3, AMS1, AMS2, AMS3, WAW2, WAW3                                |
+| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
+| Max. NIC bandwidth   	          | From 400 Mbps to 20 Gbps                                                      |
+| CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:141]:
```suggestion
| CPU type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 28]
```diff
@@ -0,0 +1,175 @@
...
+| CPU architecture  	            | `amd64`                                                                     	|
+| Resources   	                  | Dedicated vCPUs*                                                              |
+| Security feature                | Secure Encrypted Virtualization                                               |
+| Sizing   	                      | From 2 to 96 vCPUs  <br /> From 8 GiB to 384 GiB RAM                          |
+| vCPU:RAM ratio                  | 1:4                                                                           |
+
+\* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:125]: Do we add a full stop at the end of this sentence? (same for the one below)

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 29]
```diff
@@ -0,0 +1,175 @@
...
+## Workload-Optimized Instances
+
+See below the technical specifications of Workload-Optimized Instances:
+
+* High-Memory: Designed for RAM-intensive usages, and high memory production applications, these machines provide more RAM than vCPU.
+* High-CPU: Made for high computing workloads and compute-bound applications, this machine provides more vCPU than RAM.
+
+| Range                         	| Workload-Optimized                                                            |
+|:--------------------------------|:------------------------------------------------------------------------------|
+| Instance Type   	              | POP2-HM, POP2-HC                                                      	      |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:162]:
```suggestion
| Instance type   	              | POP2-HM, POP2-HC                                                      	      |
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 30]
```diff
@@ -0,0 +1,175 @@
...
+| Availability Zone   	          | PAR1, PAR2, PAR3, AMS1, AMS2, WAW2, WAW3                                      |
+| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
+| Max. NIC bandwidth   	          | From 400 Mbps to 12.8 Gbps                                                    |
+| CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```
Comment (jcirinosclwy) [inline compute/instances/reference-content/instances-datasheet.mdx:166]:
```suggestion
| CPU type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
```

**Review by jcirinosclwy — APPROVED** [review]

File: compute/instances/reference-content/instances-datasheet.mdx (hunk 28 above)
Comment (bene2k1) [reply compute/instances/reference-content/instances-datasheet.mdx:125]:
```suggestion
\* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications.
```

File: compute/instances/reference-content/instances-datasheet.mdx [hunk 31]
```diff
@@ -0,0 +1,175 @@
...
+| Availability Zone   	          | PAR1, PAR2, PAR3, AMS1, AMS2, AMS3, WAW2, WAW3                                |
+| Storage   	                    | Block (from 10 GB to 10 TB)                                                   |
+| Max. NIC bandwidth   	          | From 400 Mbps to 20 Gbps                                                      |
+| CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
+| CPU architecture  	            | `amd64`                                                                     	|
+| Resources   	                  | Dedicated vCPUs*                                                              |
+| Sizing   	                      | From 2 to 96 vCPUs  <br /> From 8 GiB to 384 GiB RAM                          |
+| vCPU:RAM ratio                  | 1:4                                                                           |
+
+\* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications
```
Comment (bene2k1) [inline compute/instances/reference-content/instances-datasheet.mdx:147]:
```suggestion
\* Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications.
```

**Pull request merged**